api/tests/python/services/name_request/test_abstract_name_request.py

```diff
@@ -0,0 +1,47 @@
+"""Tests for AbstractNameRequestMixin."""
+import datetime
+
+import pytest
+from dateutil.tz import gettz
+
+from namex.services.name_request import NameRequestService
+from namex.models import Request as RequestDAO
+
+nr_svc = NameRequestService()
+pacific_tz = gettz('US/Pacific')
+utc_tz = gettz('UTC')
+
+
+@pytest.mark.parametrize('input_datetime_utc,expected_date_utc', [
+    (datetime.datetime(2021, 1, 28, 14, 11, 00, tzinfo=utc_tz), datetime.datetime(2021, 3, 26, 6, 59, 0, tzinfo=utc_tz)),
+    (datetime.datetime(2021, 1, 28, 11, 11, 00, tzinfo=utc_tz), datetime.datetime(2021, 3, 25, 6, 59, 0, tzinfo=utc_tz)),
```

Comment:
I think it would be better to test edge cases instead of 14:11:00 and 11:11:00, since these won't catch off-by-7 or off-by-8 errors.

In fact, I don't understand why lines 16 and 17 don't resolve to the same date ???

Reply:
>In fact, I don't understand why lines 16 and 17 don't resolve to the same date ???

Yes, you're right. As I wasn't checking any dates the test was ok but yes, there was an issue because I was adding days with UTC. I fixed that and added tests to check the edge cases. THANKS!

Reply:
The new tests look great !!!